static/usage/v7/item/inputs/vue.md

```diff
@@ -44,23 +41,31 @@
   </ion-item>
 
   <ion-item>
-    <ion-label>Toggle</ion-label>
-    <ion-toggle slot="end"></ion-toggle>
+    <ion-toggle> Toggle </ion-toggle>
   </ion-item>
 
   <ion-item>
-    <ion-label>Checkbox</ion-label>
-    <ion-checkbox slot="end"></ion-checkbox>
+    <ion-checkbox> Checkbox </ion-checkbox>
   </ion-item>
 
   <ion-item>
-    <ion-label position="stacked">Range</ion-label>
-    <ion-range></ion-range>
+    <ion-range label-placement="start">
+      <div slot="label">Range</div>
```

Comment:
Yes, because `ion-range` has other slots, so we need to use a named slot in order to detect the label. The design doc that states named slots should also have a `label` property is new. Those changes have not been implemented yet.

Reply:
I'm confused. Are you saying this isn't supported yet but we intend to support it:

```
<!-- Passing plain text in a property -->
<ion-range label="My Label"></ion-range>
```

Shouldn't we be recommending this way?